src/main/java/org/betonquest/betonquest/modules/updater/source/implementations/GitHubReleaseSource.java

```diff
@@ -16,13 +16,18 @@
  */
 public class GitHubReleaseSource extends UpdateSource implements ReleaseUpdateSource {
 
+    /**
+     * The basic url for the api of GitHub.
+     */
+    public static final String RELEASES_URL = "/releases";
```

Comment:
To the above I would then also adjust here
from "RELEASES_URL" to "RELEASES_PATH"